### PR TITLE
Sign collection and version

### PR DESCRIPTION
**Pull request by MilanPospisil**
Sign collection version and sign collection actions.

Additionaly:
 - some operations has bad row key, it was missing collection namespace
 - display_signatures is now implented, when off, sign state is not displayed

It also contains upload signature, for insights - this code can be used when insights is developed.

### CODE REVIEW

**Review by nixocio — COMMENTED** [review]
@MilanPospisil overall good job. Is there a related issue associated with this change?

File: frontend/hub/api/utils.tsx [hunk 1]
```diff
@@ -14,6 +14,43 @@ import {
   putHubRequest,
 } from './request';
 
+// returns the preferred distribution base_path given a repo name
+// if there is a distribution with the same name as the repository, it will be used (as long as it's connected to the right repo too)
+// if not, the oldest will be used
+// reject if no distributions or repository
+// optional pulp_href param skips repo lookup
+
+export async function getRepositoryBasePath(name: string, pulp_href?: string) {
```
[the remark this answers is not in the document]
Comment (MilanPospisil) [reply frontend/hub/api/utils.tsx:23]: Agreed, I wanted to discuss this in the review. Im always against copy pasting. So we should have two versions - hook that uses more generaly usable function getRepositoryBasePath.

**Comment by ZitaNemeckova** [comment]
Tested in UI. Works for me. Please change the Upload button in the dialog from red to blue :)

**Review by nixocio — APPROVED** [review]
Good job @MilanPospisil please add the related issue.

**Pull request merged**